.travis.yml

```diff
@@ -1,42 +1,39 @@
-﻿dist: trusty
-sudo: false
-env:
-  global:
-    - MAKEFLAGS="-j 2"
+﻿os: linux
```

Comment:
I'm all for conventions/strong default over configuration.

I used the [travis configuration explorer](https://config.travis-ci.com/explore) and it raised some info/warnings regarding missing flags.

Still we have 2 options:
- Follow Travis default and use oldest maintained LTS version of ubuntu for stability, which might change once Xenial is EOL. 
- Stick to the latest version and upgrade manually once we feel like it.

I'm leaning towards using default and if something break (which I don't think will) we can fix it afterwards, instead of having to bump the distribution (task that nobody cares about, and will be forgotten).

So OK, I'll remove them.

Reply:
Actually both messages are infos, not warnings. 
```
[info] on root: missing os, using the default "linux"
[info] on root: missing dist, using the default "xenial"
```
So I am sure it's ok to use defaults here.

Reply:
Yup, that's what I though after pasting the configuration again in the configuration tool.